portfolio/index.html

```diff
@@ -0,0 +1,279 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio_1</title>
+    <link rel="icon" type="image/x-icon" href="icons/favicon.ico">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;700&family=Merriweather:wght@400;700&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="wrapper">
+        <header>
+            <div>
+                <img class="logo" src="icons/header_logo.svg" alt="camera image">
+                <nav>
+                    <ul>
+                        <li><a href="#skills">Skills</a></li>
+                        <li><a href="#portfolio">Portfolio</a></li>
+                        <li><a href="#video">Video</a></li>
+                        <li><a href="#price">Price</a></li>
+                        <li><a href="#contacts">Contacts</a></li>
+                        <li class="switch_lng"><span>en </span> / ru</li>
+                    </ul>
+                </nav>
+            </div>
+            <section class="first_section">
```

Comment:
Bad approach to name c;lasses according to order (order can be changed).
Better "hire_me", for example.